### ISSUE TITLE
 "INFO: StateIdle" 

### ISSUE THREAD

**Issue by flyleaf61** [norecha/pokubot#51]
how can i fix this "INFO: StateIdle"

**Comment by jarredhawkins** [comment]
Is your resolution set right? Read the instructions.

If it is, please provide some more details.

**Comment by motodemoemail** [comment]
[03.22.15 2:54:33 PM] INFO: Setting up ConfigUtils... 
[03.22.15 2:54:33 PM] INFO: Make sure in-game language is English. 
[03.22.15 2:54:33 PM] INFO: Found barracks coordinates <683, 320> 
[03.22.15 2:54:33 PM] INFO: Setting up BlueStacks App Player window... 
[03.22.15 2:54:33 PM] INFO: Setting up BlueStacks App Player resolution... 
[03.22.15 2:54:33 PM] INFO: Setting up RobotUtils... 
[03.22.15 2:54:33 PM] INFO: Setting up Barracks... 
[03.22.15 2:54:33 PM] INFO: Setup is successful. 
[03.22.15 2:54:33 PM] INFO: Click start to run. 
[03.22.15 2:54:35 PM] INFO: Starting disconnect detector... 
[03.22.15 2:54:35 PM] INFO: Running disconnect detector... 
[03.22.15 2:54:35 PM] INFO: StateIdle 
[03.22.15 2:54:36 PM] INFO: StateIdle 
[03.22.15 2:54:37 PM] INFO: StateIdle 
[03.22.15 2:54:38 PM] INFO: StateIdle 
[03.22.15 2:54:39 PM] INFO: StateIdle 
[03.22.15 2:54:40 PM] INFO: StateIdle 
[03.22.15 2:54:40 PM] WARNING: runner is cancelled. 
[03.22.15 2:54:40 PM] INFO: Interrupted by user. 
[03.22.15 2:54:40 PM] SEVERE: dc checker: sleep interrupted 

I am having this very same issue. Running Win 7 64 bit and Java version 8 update 40 v8.0.400. I am new to this, so I installed Bluestacks yesterday, and the Pokubot newest version, but still continue to have the same issue. I made the changes in the registry as well (Depth = 16 - Fullscreen = 0 - HideBootProgress = 1 - WindowState = 1 - GuestHeight/WindowHeight = 720 - GuestWidth/WindowWidth = 860). Game is in English as well.
